static/.htaccess

```diff
@@ -8,6 +8,10 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# Modeler start instance and deploy page renaming and merging
+RewriteRule ^docs/components/modeler/web-modeler/start-instance/(.*)$ /docs/components/modeler/web-modeler/run-or-publish-your-process/$1 [R=301,L]
+RewriteRule ^docs/components/modeler/web-modeler/save-and-deploy/(.*)$ /docs/components/modeler/web-modeler/run-or-publish-your-process#deploy-a-process/$1 [R=301,L]
```

Comment:
```suggestion
RewriteRule ^docs/next/components/modeler/web-modeler/start-instance/?$ /docs/next/components/modeler/web-modeler/run-or-publish-your-process/ [R=301,L]
RewriteRule ^docs/next/components/modeler/web-modeler/save-and-deploy/?$ /docs/next/components/modeler/web-modeler/run-or-publish-your-process/#deploy-a-process [R=301,L,NE]
```

Ok, a few things about these rules that I learned! This is a lot, but I want to share _why_ I'm making these recommendations.

1. I'm guessing the reason we haven't been able to redirect to page section headers is that the server encodes the `#` character as `%23`. I noticed this when running the redirects locally. 
2. But I learned that [we can prevent this by adding the `NE` flag to the rule](https://serverfault.com/a/987814). (`NE` stands for "no escape" which is very relatable 😅 ) 
3. These redirects are both incorrect for the current PR, as the changes are only made in the `next` version. So they should both be matching and targeting the `next` URLs, instead of the current version (which does not specify a version in the path). 
4. There are also issues with the wildcard matching. For the first rule, the rule works, though it could be improved to eliminate one redirect. The rule matches `start-instance/(.*)`, which means the start-instance page, _and any of its children_. This page doesn't have any children, so that's a minor point, it will just never affect anything but the main page. But there's no need to capture this group, or preserve it. 

   In reality, what we probably mean to do, is capture an optional `/` at the end of the path -- so for either `start-instance/` or `start-instance` (no `/`), redirect to the new path, with a `/` at the end. This saves the user a hop when they request `start-instance` (no `/`). Instead of getting redirected to `start-instance/` before getting redirected to `run-or-publish-your-process`, they'd go directly to the target right away. 

   All this to say, the left side should change from `...start-instance/(.*)$` to `...start-instance/?$`, and the right side to `...run-or-publish-your-process/`. 
5. The second rule is more problematic, due to including an anchor. It has the same problems described in number 4 above, but even with the `NE` flag, it redirects to an invalid URL -- `run-or-publish-your-process/#deploy-a-process/`. Note the extra slash at the end. That URL doesn't exist.

   But if we make the same changes as in number 4 -- make the left side end in `/?$` and remove the `$1` from the right side -- it goes to the correct place. A page named `run-or-publish-your-process`, and an anchor named `deploy-a-process`. 
6. This raises one last question from me - should that first rule _also_ be redirecting to a page anchor? If so, that's something I'm not accounting for in these recommendations.

Sorry for being so wordy, thanks for coming to my TED talk!

Reply:
@pepopowitz this is great! Thank you. It will save a lot of thought work for future restructurings. Would it make sense to document your findings in the documentation guide?

Reply:
Ah, it seems like you already did! https://github.com/camunda/camunda-platform-docs/pull/2097